.github/workflows/npm-publish.yml

```diff
@@ -15,7 +15,6 @@ jobs:
       - run: npm test
       - uses: JS-DevTools/npm-publish@v2
         with:
-          token: 'some_dummy_token'
-          dry-run: true
+          token: ${{ secrets.NPM_TOKEN }}
```

Comment:
@amcdnl this needs to be added in the github secrets/env variable under repository settings.